environment.yml

```diff
@@ -10,3 +10,4 @@ dependencies:
   # Optional dependencies
   - geopandas=1.0.1
   - jupyter-book=1.0.2
+  - rioxarray=0.17.0
```

Comment:
> I cloned the repository and tested it on my local machine. It seems that rioxarray needs to be included in the environment.yml file. Is this correct, or am I missing something?

Yes, added here already :smile:

Reply:
Hello @weiji14,

I’ve tested everything here, and I was planning to add a few more explanations, especially about the pros and cons of data resolution and computational requirements for large mosaics. Should I make these changes directly in the tut5_topography.md file, or should I create a new branch?

Reply:
Yes, feel free to push changes directly (be sure to do a `git pull` first in case I push any changes too :wink:). I'll be adding more text to the Antarctica section too later.